lib/metanorma/compile_validate.rb

```diff
@@ -29,31 +29,33 @@ def load_flavor(stdtype)
         Util.log("[metanorma] Info: Loading `#{flavor}` gem "\
                  "for standard type `#{stdtype}`.", :info)
       end
-      require_flavor(flavor, stdtype)
+      require_flavor(flavor)
       unless @registry.supported_backends.include? stdtype
         Util.log("[metanorma] Error: The `#{flavor}` gem "\
                  "still doesn't support `#{stdtype}`. Exiting.", :fatal)
```

Comment:
"still doesn't support `#{stdtype}`." => "does not support the standard type `#{stdtype}`."

Reply:
Fixed